src/assets/translations/en/main.json

```diff
@@ -513,8 +513,6 @@
     "noAccountsOpportunities": "You have no accounts for this asset, so staking opportunities are currently unavailable.",
     "clickHereToEarn": "Click here to earn %{apr} APR* by farming your ETH-FOX liquidity pool tokens",
     "overview": "Overview",
-    "myPositions": "My Positions",
-    "myPositionsBody": "A list of your currently active positions in DeFi.",
```

Comment:
blocking: revert me

Reply:
Done in [8db94c1](https://github.com/shapeshift/web/pull/9086/commits/8db94c1ebd64971e4a6f3dd422478112132f863f)